en/12_Project_Governance/08_Fixed_dependencies.md

```diff
@@ -22,7 +22,6 @@ Dependency | Fixed major release line
 [`monolog/monolog`](https://packagist.org/packages/monolog/monolog) | 3
 [`phpunit/phpunit`](https://packagist.org/packages/phpunit/phpunit) | 9
 [`symfony/*`](https://packagist.org/packages/symfony/) | 6
-[`webonyx/graphql-php`](https://packagist.org/packages/webonyx/graphql-php) | 15
```

Comment:
This is still a fixed dependency because we're still supporting graphql.

Reply:
Added back in